src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -18,11 +18,44 @@ import { UrlPathTemplate } from '../../../utils/urlPaths';
 import { authenticatedApiRequest } from '../../../api/apiRequest';
 import { getValueByKey } from '../../../utils/user-helpers';
 import { fetchPerson } from '../../../api/cristinApi';
+import { postOrcidCredentials } from '../../../api/orcidApi';
+import { OrcidCredentials } from '../../../types/orcid.types';
 
 interface UserOrcidProps {
   user: User;
 }
 
+function allPropertiesAreInCredentialsObject(rawCredentials: { [p: string]: string }) {
+  return (
+    rawCredentials.hasOwnProperty('expires_in') &&
```

Comment:
Du mangler kanskje ESLint extenstion i editoren?
```suggestion
    rawCredentials['expires_in'] &&
```
![bilde](https://user-images.githubusercontent.com/6313468/236171267-5715c454-f201-4b99-98a7-7efb53685d78.png)

Reply:
Npm install fikset issuet. Hadde gammel versjon av prettier.